website/docs/docs/build/packages.md

```diff
@@ -366,3 +367,18 @@ packages:
 ```
 
 </File>
+
+### About dependencies.yml
```

Comment:
I don't think we need this as an entire section. I would rather we say this like:
- Starting in v1.6, `packages.yml` has been renamed to `dependencies.yml`
- However, if you need to be using Jinja within your packages config — such as an environment variable for your private package — you need to keep using `packages.yml` for your `packages` spec for now. Implication is: better answer for private packages coming in the future.